src/lib/utils.ts

```diff
@@ -183,6 +183,22 @@ export async function updateMetadata(ctx: any, network: number) {
   }
 }
 
+export async function getCurrentGasPrices() {
+  const {
+    data: { fast, fastest, safeLow },
+  } = await axios(constants.ETH_GAS_STATION_URL)
+  return {
+    fast: fast / 10,
+    fastest: fastest / 10,
+    safeLow: safeLow / 10,
```

Comment:
`average` is missing here. `safeLow` is the one I usually exclude, since the transactions submitted with that price can often take 10 - 20 mins.